src/elm/Session/LoggedIn.elm

```diff
@@ -631,11 +631,13 @@ isActive page route =
 
 viewFooter : Shared -> Html msg
 viewFooter _ =
-    footer [ class "bg-white w-full flex flex-wrap mx-auto border-t border-grey p-4 pt-6 h-40 bottom-0" ]
-        [ p [ class "text-sm flex w-full justify-center items-center" ]
-            [ text "Created with"
-            , Icons.heart
-            , text "by Satisfied Vagabonds"
+    footer [ class "bg-white w-full flex flex-wrap justify-center mx-auto border-t border-grey p-4 pt-6 h-40 bottom-0" ]
+        [ div [ class "flex justify-around " ]
+            [ p [ class "text-sm flex w-full justify-center text-center" ]
+                [ text "Created with" ]
+            , div [ class "w-1/5" ] [ Icons.heart ]
+            , p [ class "text-sm flex w-full justify-center text-center" ]
+                [ text "by Satisfied Vagabonds" ]
```

Comment:
It looks like these changes mess up the footer design...

Here's the footer from Figma (any page):
![image](https://user-images.githubusercontent.com/140053/87656243-e7a00300-c761-11ea-849b-c24f56a148e7.png)

Here's the footer from this branch:
![image](https://user-images.githubusercontent.com/140053/87656307-fc7c9680-c761-11ea-8762-7b57bf5d4aa4.png)